build/automation/var/profile/demo.mk

```diff
@@ -10,6 +10,7 @@ DEPLOYMENT_SECRETS = $(SERVICE_PREFIX)/deployment
 
 K8S_TTL =
 API_HOST = uec-dos-api-cs
+REQ_PER_SEC_PROXY_THROTTLE = 200
```

Comment:
We have changed the request per second value from 3 to 200. Is this intentional? Originally, we decided to keep that value low to protect DoS from any risk of overloading it with requests.

Reply:
We were testing the limit settings of the pods.  We've set this back to 3 and added ADR-009 to explain this